src/debug.js

```diff
@@ -1,7 +1,5 @@
 import debug from "debug";
 
-debug.enable("github-semantic-version:*");
-
```

Comment:
The one question I do have is since `debug.js` is imported across multiple files, I couldn't think of a great way to enable/disable based on the `--debug` flag. I removed this line and left it up to using the `DEBUG=github-semantic-version:*` environment variable before running the app. Is there a better way?